### PR TITLE
Update command docs with latest labels

### PR DESCRIPTION
**Pull request by jandonyada**

### CODE REVIEW

File: workbot/workbot-actions.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Workbot actions allow Workbot to post notifications into a specified channel whe
 Workbot supports 3 actions:
 * [Post command reply](#post-command-reply)
 * [Post notifications](#post-notifications)
+* [Post messages](#post-messages)
 * [Download attachment](#download-attachment)
 
 ## Post command reply
```
Comment (zweedao) [inline workbot/workbot-actions.md]: post-command-reply2.png is too small, can't see the text well. Should cut the blank part on the right to make the left part bigger.

File: workbot/workbot-actions.md [hunk 2]
```diff
@@ -136,17 +135,26 @@ The following table lists the fields available in a **Post command reply** actio
...
+Related commands are deprecated-->
     </tbody>
 </table>
 
+## Post messages
+This action will post a message response to a user who invokes a command. By default, Workbot posts direct messages only to the user who installed Workbot. Note, it is also possible to subscribe from a channel (where Workbot participates) to these notifications.
+
+The fields available are simillar to that of the **Post command reply** action, with all the typical fields you see in a Slack message. However, there is the advanced section with 2 fields: **Message to update** and **Thread ID**.
+
+**Message to update** allows you to update a previously posted message using the <code>Message ID</code> datapill from a **Command** trigger from a **Post message** or **Post notification** from an earlier action step.
```
Comment (zweedao) [inline workbot/workbot-actions.md]: This sentence is quite long and has multiple 'from'. Not easy to understand here. Should be broken down into shorter & more simple sentences.

File: workbot/workbot-actions.md [hunk 3]
```diff
@@ -136,17 +135,26 @@ The following table lists the fields available in a **Post command reply** actio
...
+
+**Thread ID** allows you to post a message within an existing thread using the <code>Thread ID</code> datapill from a **Post message** or **Post notificaiton** from an earlier action step.
```
Comment (zweedao) [inline workbot/workbot-actions.md]: This sentence is also quite long. Should be broken down into more simple sentences.

File: workbot/workbot-actions.md (hunk 2 above)
Comment (zweedao) [reply workbot/workbot-actions.md]: Screenshot here will make it easier to understand. I don't quite understand this feature from reading this.

File: workbot/workbot-actions.md (hunk 3 above)
Comment (zweedao) [reply workbot/workbot-actions.md]: Screenshot here will make it easier to understand. I don't quite understand this feature from reading this.

File: workbot/workbot-triggers.md [hunk 4]
```diff
@@ -33,36 +33,36 @@ The following table lists the configurable input fields in the trigger, and what
     </thead>
     <tbody>
         <tr>
-            <td>Application</td>
-            <td>1st part of the Workbot command. The app the Workbot command is working with. Groups the command under this application (when user types app name, command will show up as a button).
+            <td>App</td>
+            <td>1st part of the Workbot command. The app the Workbot command is working with. Groups the command under this application (when user types the app's name, command will show up as a button).
```
Comment (zweedao) [inline workbot/workbot-triggers.md]: The app "that" Workbot command is working with...

Comment (zweedao) [reply workbot/workbot-triggers.md]: A specific example here will make it easier to imagine. e.g. Salesforce

File: workbot/workbot-triggers.md [hunk 5]
```diff
@@ -33,36 +33,36 @@ The following table lists the configurable input fields in the trigger, and what
...
             <td>
               3rd part of the Workbot command. The record that the Workbot command works with, e.g. invoice, customer, ticket.
             </td>
         </tr>
         <tr>
-            <td>Input parameters</td>
+            <td>Command input fields</td>
             <td>
-              The data to ask from the user so as to successfully carry out recipe actions, defined in a specific format. Parameters may have the type file, which takes as input file content (e.g. text, binary content), that you may upload into Slack. You can use this file in subsequent operations using the Workbot action <a href="/workbot/workbot-actions.html">Download Attachment.</a>
+              The input to ask from the user so as to successfully carry out recipe actions, defined in a specific format. Parameters may have the type <code>file</code>, which takes in input file content (e.g. text, binary content) as input, which you may upload into Slack. You can use this file in subsequent operations using the Workbot action <a href="/workbot/workbot-actions.html">Download Attachment.</a>
```
Comment (zweedao) [inline workbot/workbot-triggers.md]: ..., which accepts file content (e.g. text, binary content) as input, ...

File: connectors/slack.md [hunk 6]
```diff
@@ -32,8 +32,8 @@ Authorize Workato to access your Slack organization via the OAuth2 standard. Thi
 ### Using the post message action
 The **Post message** action can be used to post a message to a channel or a direct message to a Slack user. By default, the message will be posted as the Workato app, but it can be configured to post under another name and corresponding icon image.
 
-#### Example message with attachment
-The following shows an example of how the different fields show up in Slack. As the **Post message as** and **Icon image URL** input fields have not been configured, the message displays as being posted by the Workato app.
+#### Example with message attachments
+The following shows an example of how the different fields show up in Slack. As the **Post message** and **Image URL** input fields have not been configured, the message displays as being posted by the Workato app.
 
 ![Post message example with attachment](/assets/images/connectors/slack/post-message-basic-example.png)
```
[the remark this answers is not in the document]
Comment (jandonyada) [reply connectors/slack.md]: Updated

File: workbot/workbot-actions.md (hunk 1 above)
Comment (jandonyada) [reply workbot/workbot-actions.md]: Will be updated

File: workbot/workbot-actions.md (hunk 2 above)
Comment (jandonyada) [reply workbot/workbot-actions.md]: Re-worded and added screenshots for both message to update and thread ID

File: workbot/workbot-actions.md (hunk 3 above)
Comment (jandonyada) [reply workbot/workbot-actions.md]: Re-worded and added screenshots for both message to update and thread ID

File: workbot/workbot-triggers.md (hunk 4 above)
Comment (jandonyada) [reply workbot/workbot-triggers.md]: Will be updated. Also added some styling for the examples.

Comment (zweedao) [reply workbot/workbot-triggers.md]: did you forget to delete this part? Duplicated with line 169-202

Comment (jandonyada) [reply workbot/workbot-triggers.md]: Hmm no, it's similar but not the same - the first table is the input, and the 2nd table is the output. App, Action and Action data are in both the input and the output.

File: connectors/slack.md [hunk 7]
```diff
@@ -35,7 +35,7 @@ The **Post message** action can be used to post a message to a channel or a dire
 #### Example message with attachment
 The following shows an example of how the different fields show up in Slack. As the **Post message as** and **Icon image URL** input fields have not been configured, the message displays as being posted by the Workato app.
 
-![Post message example with attachment](/assets/images/connectors/slack/post-message-basic-example.png)
+![Post message example with attachment](/assets/images/connectors/slack/post-message-basic-example-2.png)
```
Comment (eeshansim) [inline connectors/slack.md]: so is post-message-basic-example.png used anywhere else? Can we just replace the image?

File: connectors/slack.md [hunk 8]
```diff
@@ -93,7 +93,7 @@ The danger styled buttons, when clicked, generates a popup prompt that asks the
 
 The prompt that pops up can be completely configured in the button configuration in the Slack action **Post message**.
 
-![Popup prompt example](/assets/images/connectors/slack/popup-prompt-example.png)
+![Popup prompt example](/assets/images/connectors/slack/popup-prompt-example-2.png)
```
Comment (eeshansim) [inline connectors/slack.md]: Same here

File: workbot/workbot-actions.md [hunk 9]
```diff
@@ -9,14 +9,15 @@ Workbot actions allow Workbot to post notifications into a specified channel whe
...
+* [Post messages](#post-messages)
 * [Download attachment](#download-attachment)
 
 ## Post command reply
 Post command reply allows you to customize how Workbot replies when an event is completed. This reply can be a simple message about the task completion, or a prompt for the user to take a subsequent action once the first has been done, e.g. after retrieving data for a custom account in Salesforce, ask if the user wishes to copy that information across to another application.
 
 Here is what the Post Command Reply action's output will look like in Slack.
 
-![Workbot post command reply](/assets/images/workbot/workbot-custom-commands/post-command-reply2.png)
+![Workbot post command reply](/assets/images/workbot/workbot-custom-commands/post-command-reply3.png)
```
Comment (eeshansim) [inline workbot/workbot-actions.md]: Same here

File: workbot/workbot-actions.md [hunk 10]
```diff
@@ -136,17 +135,29 @@ The following table lists the fields available in a **Post command reply** actio
...
+This action will post a message response to a user who invokes a command. By default, Workbot posts direct messages only to the user who installed Workbot. Note, it is also possible to subscribe from a channel (where Workbot participates) to these notifications.
+
+The fields available are simillar to that of the **Post command reply** action, with all the typical fields you see in a Slack message. However, there is the advanced section with 2 fields: **Message to update** and **Thread ID**.
+
+**Message to update** allows you to overwrite a previously posted message from an earlier action step. Simply use the <code>Message ID</code> datapill from a **Post message** or **Post notification** output datatree.
+![Message to update example](/assets/images/workbot/workbot-actions/message-to-update-example.png)
+*Message to update example*
+
+**Thread ID** allows you to post a message within an existing thread in Slack. Simply use the <code>Thread ID</code> datapill from a **Post message** or **Post notification** output datatree.
+![Thread ID example](/assets/images/workbot/workbot-actions/thread-id-example.png)
```
Comment (eeshansim) [inline workbot/workbot-actions.md]: missing space after image

**Pull request merged**